src/components/AuthForm.js

```diff
@@ -20,10 +20,24 @@ function AuthForm (props) {
       }
     }
 
-    const urlForUsers = endpointApiUrl.concat("/","users")
+    const urlForUsers = endpointApiUrl.concat("/users")
   
     let register = await axios.post(urlForUsers, data)
-    let tokenInfo = sessionStorage.setItem('jwt', register.data.user.token);
+    
+    sessionStorage.setItem('jwt', register.data.user.token);
+  }
+
+  async function submitSignInData() {
+    const data = {
+      "user": {
+      email,
+      password
+      }
+    }
+    const urlForLogIn = endpointApiUrl.concat("/users/login")
```

Comment:
Take a look at template strings. I think it makes the code more readable when having string interpolation.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
It indeed looks way simpler and easier to read with template strings! All necessary codes have been updated to use template strings. Thanks!